lib/compute/agent-nodes.ts

```diff
@@ -142,7 +142,7 @@ export class AgentNodes {
       numExecutors: 2,
       amiId: 'ami-09f55bf0827296c51',
       initScript: 'sudo dnf clean all && sudo rm -rf /var/cache/dnf && sudo dnf repolist &&'
-        + ' sudo dnf update --releasever=latest --skip-broken --exclude=openssh* --exclude=docker* --exclude=gh* --exclude=python* -y && docker ps',
```

Comment:
Do we want to include python now?

Reply:
Python* is removed as it is not broken anymore.